test/__tests__/solidity/gatherSources.spec.ts

```diff
@@ -23,65 +25,64 @@ function deepSubset<T>(a: T[], b: T[]): boolean {
   return a.every(obj1 => b.some(obj2 => deepequal(obj1, obj2)));
 }
 
-describe("gatherSources function", function() {
-  const resolver: ResolverEngine<ImportFile> = SolidityImportResolver();
-
-  beforeAll(function() {
-    process.chdir(__dirname);
-  });
+// when using mock fs, we are being thrown into the root of the filesystem
+// we need to call it so process.cwd() makes sense
+process.chdir(__dirname);
```

Comment:
This is called in wrong phase - this will chdir ALL the tests, not only this file. Move it into `beforeAll()`

Reply:
If you want data to use cwd, you can set it in beforeAll too

Reply:
'data' has to be defined, not only declared, globally. It uses '__dirname' now, instead of 'process.cwd()'.